### DISCUSSION TITLE
📚 🔎 Ryzen Threadripper 3990X + 128GB RAM + (2+2+1)TBs NVMe (Firecuda 520 series)

### DISCUSSION THREAD

**Discussion by brunobrdo** [madMAx43v3r/chia-plotter#179] [General]
I'm buying some hardwares just to create plots.

2 questions:
[1] What could be the expected time per plot with my new PC configuration?
[2] But now I'm wondering: To improve the time per plot, what should I buy?

**Comment by brunobrdo** [comment]
Anyone can help me?

**Reply by tobros91** [reply]
You might want to get more ram,

Check out my post here with threadripper 3970x

https://github.com/madMAx43v3r/chia-plotter/discussions/92#discussioncomment-859881
